package.json

```diff
@@ -31,6 +31,7 @@
     "@types/uuid": "^3.4.8",
     "@types/ws": "^6.0.4",
     "aws-sdk": "^2.640.0",
+    "bl": "^2.2.1",
```

Comment:
comment that we don't need this directly, but we want this version for security

Reply:
Removed, and then let npm update deal with all dependency updates in package-lock.json for now.